docs/install/android.mdx

```diff
@@ -8,63 +8,81 @@ title: Build for Android
-| :-------------------------- | :------------------------------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| **`--ndk-path`**            | Mandatory                       | The path of Android NDK, for example: `/opt/Android/Sdk/ndk/22.0.7026061`                                                                                                             |
-| **`--target`**              | Mandatory                       | Target architecture which accepts the following values: `arm64-v8a`, `armeabi-v7a`, `x86`, `x86_64`                                                                                   |
-| **`--api`**                 | Optional. Default value is `29` | Android API level. Must be between 21 and 30                                                                                                                                          |
-| **`--libpcap-include-dir`** | Mandatory                       | `libpcap` header files directory. Can be downloaded from [the `libpcap-android` GitHub repo](https://github.com/seladb/libpcap-android)                                               |
-| **`--libpcap-lib-dir`**     | Mandatory                       | `libpcap` pre-compiled libs directory. Can be downloaded from [the `libpcap-android` GitHub repo](https://github.com/seladb/libpcap-android) for all 4 targets and API versions 21-30 |
+```shell
+cmake \
+-DCMAKE_TOOLCHAIN_FILE="${ANDROID_NDK}/build/cmake/android.toolchain.cmake" \
+-DANDROID_PLATFORM="${API_VERSION}" \
+-DANDROID_ABI="${ABI}" \
+-DPCAP_LIBRARY="${LIBPCAP_DIR}/${ABI}/${API_VERSION}/libpcap.a" \
+-DPCAP_INCLUDE_DIR="${LIBPCAP_DIR}/include/" \
+-DPCAPPP_BUILD_TESTS=OFF \
+-DPCAPPP_BUILD_EXAMPLES=OFF \
+-S . -B build
+```
 
-Here is an example output:
+:::tip NOTE
 
-```shell
-./configure-android.sh --ndk-path /opt/Android/Sdk/ndk/22.0.7026061 --target arm64-v8a --api 29 --libpcap-include-dir ~/libpcap-android/include --libpcap-lib-dir ~/libpcap-android/arm64-v8a/30
+`-DPCAPPP_BUILD_TESTS=OFF` and `-DPCAPPP_BUILD_EXAMPLES=OFF` are not mandatory, but building PcapPlusPlus tests and examples for Android usually doesn't make sense.
 
-******************************************
-PcapPlusPlus Android configuration script
-******************************************
+:::
 
-Build machine OS type: linux
-Build machine OS arch: x86_64
+And then initiate the build in one of two ways:
 
-Android NDK root: /opt/Android/Sdk/ndk/22.0.7026061
-Anrdoid API level: 30
-Android toolchain path: /opt/Android/Sdk/ndk/22.0.7026061/toolchains/llvm/prebuilt/linux-x86_64
+- Using CMake:
+  ```shell
+  cmake --build build
+  ```
+- Using `make`:
+  ```shell
+  cd build
+  make
```

Comment:
make -C build if you don't want to cd

Reply:
I suggest we leave the `cd`, most people are more familiar with this